### PR TITLE
feat(react-media): rename react audio to react media

### PR DESCRIPTION
**Pull request by sayinserdar**

### CODE REVIEW

File: packages/react-media/providers/MediaProvider.tsx [hunk 1]
```diff
@@ -91,29 +30,31 @@ interface AudioProviderProps {
...
     () => shuffled[cursor],
-    [shuffled, cursor]
+    [shuffled, cursor],
   );
   const trackRef = useSynchronizedRef(track);
 
-  const element = useMemo(
-    () => (typeof Audio !== "undefined" ? new Audio() : undefined),
-    []
+  const elementRef: MaybeRef<HTMLMediaElement> =  useRef(
```
Comment (sayinserdar) [inline packages/react-media/providers/MediaProvider.tsx:54]: @thijsdaniels This part of the provider was rendering the hook conditionally and that's why it's the only difference comparing to the section library version.

**Review by thijsdaniels — APPROVED** [review]
looks good!!

File: packages/react-media/providers/MediaProvider.tsx (hunk 1 above)
Comment (sayinserdar) [reply packages/react-media/providers/MediaProvider.tsx:54]: I had a linter error.

Comment (sayinserdar) [reply packages/react-media/providers/MediaProvider.tsx:54]: Let me try to find the exact error.

Comment (sayinserdar) [reply packages/react-media/providers/MediaProvider.tsx:54]: @thijsdaniels this is what you get if you keep it the way it is in the section library.
image

**Pull request merged**